recirq/qml_lfe/util.py

```diff
@@ -0,0 +1,99 @@
+# Copyright 2021 Google
```

Comment:
I'd strongly prefer if you could come up with a more specific name than `util.py`. see go/no-utils

Reply:
Changed.